packages/no-trapping-shim/CHANGELOG.md

```diff
@@ -0,0 +1,143 @@
+# Change Log
```

Comment:
@kriskowal This first commit is the pristine state produced from 
```sh
$ ./scripts/create-package.sh no-trapping-shim
```
See the voluminous `CHANGELOG.md` in that first commit, which I truncated in a subsequent commit. Is this a tooling bug, where the thing that updated `CHANGELOG.md` in general does not know about the one that serves as a template for new packages?

Reply:
@kriskowal should I file a bug on this? Do you expect to fix it by changing the tooling for updating `CHANGELOG.md` or by changing `create-package.sh`?